src/index.js

```diff
@@ -272,11 +272,16 @@ class TrustWeb3Provider extends EventEmitter {
    */
   postMessage(handler, id, data) {
     if (this.ready || handler === "requestAccounts") {
-      window.webkit.messageHandlers[handler].postMessage({
+      let object = {
         name: handler,
         object: data,
         id: id,
-      });
+      };
+      if (window.tw && window.tw.postMessage) {
+        window.tw.postMessage(JSON.stringify(object));
```

Comment:
```suggestion
        window.trustwallet.postMessage(JSON.stringify(object));
```

let's use more expressive name `trustwallet` for the object